docs/release_notes/0.139.0.md

```diff
@@ -0,0 +1,19 @@
+# Release v0.139.0
+
+## 🚀 Features
+
+- Security Policy for eksctl project (#6541)
```

Comment:
I'm not sure if this should be under `Features`.

Reply:
Its a feature you enable for your project 
![Screenshot 2023-04-21 at 12 49 12](https://user-images.githubusercontent.com/10995114/233628592-b2166b0c-85d3-40b1-ad2e-6f08f44f7009.png)